cassandra/Dockerfile

```diff
@@ -39,24 +38,28 @@ ENV CASSANDRA_VERSION="2.2.1" \
 # Become the root user to be able to install and setup Cassandra under /opt
 USER root
 
+#
+RUN yum install -y -q bind-utils && \
+    yum clean all
+
+COPY apache-cassandra-2.2.4-bin.tar.gz.md5 \
+     /tmp
+
 # Copy the Cassandra binary to the /opt directory
 RUN cd /opt; \
     curl -LO https://archive.apache.org/dist/cassandra/$CASSANDRA_VERSION/apache-cassandra-$CASSANDRA_VERSION-bin.tar.gz && \
+    if [ "$(md5sum apache-cassandra-2.2.4-bin.tar.gz | awk '{print $1}')" != "$(cat /tmp/apache-cassandra-2.2.4-bin.tar.gz.md5)" ]; then exit 1; fi && \
```

Comment:
Maybe we should add a print statement explicitly identifying why we are exiting early